client/settings/express-checkout/express-checkout-customizer.js

```diff
@@ -0,0 +1,192 @@
+/**
```

Comment:
Could you please also copy the tests for this file?

Reply:
I believe you are referring to this file?

https://github.com/Automattic/woocommerce-payments/blob/4e52a889a39a3498ae98a3fd76412e984479f381/client/settings/payment-method-settings/test/index.js

I think I remember looking through that one and finding nothing particularly useful for Stripe.
The implementation for Stripe doesn't generate breadcrumbs, doesn't make a banner, we got rid of the methodId usage...